packages/@o3r/schematics/src/modules/modules.display.rule.ts

```diff
@@ -19,13 +40,13 @@ export function displayModuleList(keyword: string, scopeWhitelist: string[] | re
     try {
       const modules = await getAvailableModulesWithLatestPackage(keyword, {
         scopeWhitelist,
-        onlyNotInstalled: onlyNotInstalledModules,
+        onlyNotInstalled: !!options?.onlyNotInstalledModules,
         logger: context.logger,
         workspaceConfig
       });
       const message = modules
-        .filter((mod) => !onlyCmsModules || mod.keywords?.includes(OTTER_KEYWORD_CMS))
-        .map((mod) => formatModuleDescription(mod, getPackageManagerExecutor(workspaceConfig), tagMap))
+        .filter((mod) => !options?.onlyCmsModules || mod.keywords?.includes(OTTER_KEYWORD_CMS))
+        .map((mod) => formatModuleDescription(mod, getPackageManagerExecutor(workspaceConfig, options?.packageName), tagMap))
```

Comment:
should we highlight the modules not under the @o3r scope?

Reply:
Nop, the purpose of this code is to display the modules that user can installed it's project.
We do not want to display the modules that we do not want him/her to install

Reply:
What I meant is that is there any risk we could list modules out of scope and if we do - should we enhance the log to warn the log that the scope is not under our control?